test/regression/2016-11-24-bitcast-weak-alias.c

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: linux
```

Comment:
I think it would be better to add a `linux` and `darwin` feature to `config.available_features`. That way you can also write `darwin` only tests. It also better in that `XFAIL:` actually still runs the test where as the `REQUIRES:` directive skips running the test entirely.